applications/MeshMovingApplication/custom_utilities/move_mesh_utilities.h

```diff
@@ -28,20 +28,20 @@ namespace MoveMeshUtilities {
 typedef Element::GeometryType GeometryType;
 typedef Element::VectorType VectorType;
 
-void CheckJacobianDimension(GeometryType::JacobiansType &rInvJ0,
+void KRATOS_API(MESH_MOVING_APPLICATION) CheckJacobianDimension(GeometryType::JacobiansType &rInvJ0,
                             VectorType &rDetJ0, const GeometryType &rGeometry);
 
-void MoveMesh(ModelPart::NodesContainerType &rNodes);
+void KRATOS_API(MESH_MOVING_APPLICATION) MoveMesh(ModelPart::NodesContainerType &rNodes);
 
-ModelPart* GenerateMeshPart(ModelPart &rModelPart,
+ModelPart* KRATOS_API(MESH_MOVING_APPLICATION) GenerateMeshPart(ModelPart &rModelPart,
```

Comment:
@roigcarlo it seems to not like this function, it works neither with nor without `KRATOS_API`
any idea?

Reply:
maybe it just doesn't work with such types, I could refactor the function...

Reply:
The * is the problem, it has to be either:
```C++
KRATOS_API(MESH_MOVING_APPLICATION) ModelPart* GenerateMeshPart
```
```C++
// Please don't use this one.
ModelPart KRATOS_API(MESH_MOVING_APPLICATION) * GenerateMeshPart
```
or 
```C++
typedf ModelPart* ModelPartPtr;
ModelPartPtr KRATOS_API(MESH_MOVING_APPLICATION) GenerateMeshPart
```

Reply:
cool thx I try

Reply:
worked, thx